portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/controlpanel/apps/appmanager/AppManagement.testcase

```diff
@@ -552,6 +552,37 @@ definition {
 			appStatus = "Active");
 	}
 
+	@description = "This test assert that user can filter app by Price filter in marketplace store."
```

Comment:
assert should be asserts
This test asserts that user can filter app by Price in marketplace store.

Reply:
Done